Use same manifest for upstr and downstr servers; add relevant links (#3505)
theforeman/foreman-documentation: guides/common/modules/proc_configuring-projectserver-to-synchronize-content-over-a-network.adoc

.Prerequisites
* A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.

> * You imported the subscription manifest on both the upstream and downstream {ProjectServer}. This is the same manifest that was used in xref:Importing_a_Red_Hat_Subscription_Manifest_into_Server_{context}[].

Reviewer: ```suggestion
* You imported the subscription manifest on both the upstream and downstream {ProjectServer}.
Use the same manifest.
For more information, see xref:Importing_a_Red_Hat_Subscription_Manifest_into_Server_{context}[].
```

.Prerequisites
* A network connection exists between the upstream {ProjectServer} and the downstream {ProjectServer}.

> * You imported the subscription manifest on both the upstream and downstream {ProjectServer}.
> Use the same manifest.
> For more information, see xref:Importing_a_Red_Hat_Subscription_Manifest_into_Server_{context}[].

Reviewer: ```suggestion
* You imported the same subscription manifest on both the upstream and downstream {ProjectServer}.
For more information, see xref:Importing_a_Red_Hat_Subscription_Manifest_into_Server_{context}[].
```
